integration-tests/hibernate-orm-tenancy/datasource/src/main/resources/database/discriminator/V1.0.0__create_fruits.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE known_fruits
```

Comment:
This file seems to have been added by mistake.

Reply:
It was just to keep the example as the others with Flyway and some example. I can replace all the stuff with just an import.sql and without linking this file in the documentation ?